### ISSUE TITLE
NeoVim support

### ISSUE THREAD

**Issue by tuurep** [jannis-baum/vivify.vim#3]
Hi, I was going to try creating a NeoVim version of this, since `job_start()` doesn't exist in nvim. However I noticed that the only change it needs is to drop the underscore so `job_start()` => `jobstart()` and looks like it works as expected in nvim.

Would you prefer to create a separate nvim plugin (I would be willing to maintain too) or add something like 

```
if has('nvim')
   " use 'jobstart'
else
   " use 'job_start'
endif
```

in `vivify.vim` ?

**Comment by jannis-baum** [comment]
Nice, thanks for finding a solution already! I think it'd be better to just add your suggestion to this plugin since the difference is so minor :) Feel free to open a PR and I'll merge it!

**Comment by tuurep** [comment]
Yeah I thought so, too. I already use so many plugins with "vim" in the name but it works in both. Thank you, I'll send a PR either later today, or tomorrow :)

**Comment by jannis-baum** [comment]
Closed in #4
